docs/ref/middleware.txt

```diff
@@ -320,7 +320,7 @@ you are using Django to do something like require authorization in order to
 download files and you cannot set the header using your Web server, this
 setting will be useful.
 
-.. _IE Security Blog: http://blogs.msdn.com/b/ie/archive/2008/09/02/ie8-security-part-vi-beta-2-update.aspx
+.. _IE Security Blog: https://blogs.msdn.com/b/ie/archive/2008/09/02/ie8-security-part-vi-beta-2-update/
```

Comment:
perhaps https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options would be a better reference